Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1183,6 +1183,13 @@ working, take a look at `this KEP
     free mode, make sure that default Kubernetes services like ``kube-dns`` and ``kubernetes``
     have the required label value.
 
+Outside access to ClusterIP services
+************************************
+
+By default Cilium does not route packets coming from outside the cluster to a service endpoint 
+exposed with a ``ClusterIP`` service. Routing to ``ClusterIP`` services can be enabled using the 
+by setting ``config.enableClusterIPExternalAccess=true``.
```

Comment:
Could we try to elaborate more on the rationale here on why we don't expose it by default? Back then we didn't expose it due to the assumption that the range would typically be non-routable, and given that, as a security measure, we also do not expect such traffic on the phys iface. (Plus there's NodePort which would accomplish similar things just in different IP/port range (if not backed by e.g. LoadBalancer). Perhaps there are additional pointers from K8s doc.)

Reply:
I added the link to k8s documentation about this (which we already had below so I just moved it here).